includes/why-grpc.md

```diff
@@ -0,0 +1,12 @@
+Our example is a simple route mapping application that lets clients get
+information about features on their route, create a summary of their route, and
+exchange route information such as traffic updates with the server and other
+clients.
+
+With gRPC we can define our service once in a `.proto` file and implement clients
```

Comment:
...generate clients and servers...

Reply:
Hmmm. Well, it's a bit tricky to word because Protobuf generates *scaffolding* for clients and servers that you then have to implement. But I think generates is probably more clear to the average reader here.